resources/authgear/templates/en/web/__error.html

```diff
@@ -40,6 +40,9 @@
       {{ template "error-no-passkey" (dict "type"
       $.FirstNonPasskeyPrimaryAuthenticatorType) }}
     </li>
+    <li id="error-message-passkey-empty-allow-credentials" class="hidden">
+      {{ template "error-passkey-empty-allow-credentials" (dict "variant" $.x_login_id_input_type) }}
```

Comment:
The value of `x_login_id_input_type` are `email`, `phone` and `text`. https://github.com/authgear/authgear-server/blob/d47e0223dc3e557dcb07f5bd156ad8bf0a320cdb/pkg/auth/handler/webapp/util.go#L52
So when the config is username only, the error message will be wrong.

Should we use `LoginIDInputVariant` if `x_login_id_input_type` is not `phone`, like the error `error-login-id-required`?

https://github.com/authgear/authgear-server/blob/d47e0223dc3e557dcb07f5bd156ad8bf0a320cdb/resources/authgear/templates/en/web/__error.html#L67

Reply:
I revamped this part. Now error-login-id-required is greatly simplified. error-passkey-empty-allow-credentials use the same values as error-login-id-required. They now should display the contextual login ID type.

Reply:
@carmenlau ,
@louischan-oursky has added more commits, please check when your are available. Thanks 🙏